Lib/test/test_posix.py

```diff
@@ -642,12 +642,16 @@ def test_stat(self):
 
     @unittest.skipUnless(hasattr(posix, 'mkfifo'), "don't have mkfifo()")
     def test_mkfifo(self):
-        os_helper.unlink(os_helper.TESTFN)
+        if sys.platform == "vxworks":
+            fifo_path = os.path.join("/fifos/", os_helper.TESTFN)
+        else:
+            fifo_path = os_helper.TESTFN
+        os_helper.unlink(fifo_path)
```

Comment:
It seems like this function doesn't remove the FIFO on success.

Please add this line here:

```
self.addCleanup(os_helper.unlink, fifo_path)
```

Reply:
Modified. Thanks.